src/pages/offer/PeersTable.tsx

```diff
@@ -61,18 +69,26 @@ const PeerRow: React.FC<PeerRowProps> = ({ index, peer }) => {
             <Cell>
               <A href={`/peer/${peer.id}`}>{peer.id}</A>
             </Cell>
+            {/* Current CC */}
+            <Cell>
+              {peer.currentCapacityCommitment ? (
+                <A href={`/capacity/${peer.currentCapacityCommitment?.id}`}>
+                  {formatHexData(peer.currentCapacityCommitment?.id, 15, 15)}
+                </A>
```

Comment:
no need for `?` since you already checked peer.currentCapacityCommitment is defined. Also there is a default lint rule for that
```suggestion
                <A href={`/capacity/${peer.currentCapacityCommitment.id}`}>
                  {formatHexData(peer.currentCapacityCommitment.id, 15, 15)}
                </A>
```